bootstrap_cfn/fab_tasks.py

```diff
@@ -93,7 +93,7 @@ def get_config():
 
     cfn_config = ConfigParser(project_config.config, get_stack_name())
     cfn = Cloudformation(aws_config)
-    return aws_config, cfn, cfn_config
+    return aws_config, cfn, cfn_config, project_config
```

Comment:
Hmmmm this is fast going beyond the point of readability/maintainability.

We need to come up with a better solution to this.